### PR TITLE
update release instructions

### PR DESCRIPTION
**Pull request by oliver-sanders**

### CODE REVIEW

File: docs/howto/create-a-release.md [hunk 1]
```diff
@@ -3,6 +3,63 @@
...
+* [ ] cylc-uiserver
+* [ ] metomi-rose
+* [ ] cylc-rose
+
+Misc (after the above has been completed):
+
+* cylc-doc
+  * [ ] bump instersphinx versions if required
+  * [ ] review installation instructions
+  * [ ] deploy (run deploy action, can be re-deployed if necessary)
```
Comment (MetRonnie) [inline docs/howto/create-a-release.md:54]:
```suggestion
  * [ ] deploy (run the "deploy" workflow on GitHub Actions) (can be re-deployed later if necessary)
```

**Review by MetRonnie — COMMENTED** [review]
1 suggestion

**Review by MetRonnie — APPROVED** [review]
I've also added Hilary as a reviewer as on Element he said he was a little hazy on the release process

**Comment by oliver-sanders** [comment]
Sure, will re-do this once we've closed #130

**Comment by oliver-sanders** [comment]
Added the alternative conda install command for locally built packages.

File: docs/howto/create-a-release.md [hunk 2]
```diff
@@ -144,34 +203,33 @@ configured to avoid automatically upload the package. Open your `~/.condarc`
...
+# This will take some minutes and print useful information.
+conda build recipe/
+# The following command will install the locally created package. Before
+# installing it will ask you to confirm. Scroll up and search the
+# package name. The right-side column must show a location like
+# .../anaconda3/conda-bld/linux-64::cylc-uiserver-0.1-py37_1.
+# This confirms you are installing the local build. Here $PACKAGE_NAME
+# could be something like cylc-uiserver.
+conda install --use-local $PACKAGE_NAME
+# If the last command fails, try:
```
Comment (MetRonnie) [inline docs/howto/create-a-release.md:231]:
```suggestion
# If the last command doesn't use the right build, try:
```

**Review by hjoliver — APPROVED** [review]
:+1: super helpful

File: docs/howto/create-a-release.md [hunk 3]
```diff
@@ -144,34 +203,33 @@ configured to avoid automatically upload the package. Open your `~/.condarc`
...
+conda build recipe/
+# The following command will install the locally created package. Before
+# installing it will ask you to confirm. Scroll up and search the
+# package name. The right-side column must show a location like
+# .../anaconda3/conda-bld/linux-64::cylc-uiserver-0.1-py37_1.
+# This confirms you are installing the local build. Here $PACKAGE_NAME
+# could be something like cylc-uiserver.
+conda install --use-local $PACKAGE_NAME
+# If the last command doesn't use the right build, try:
+conda install -c ${CONDA_PREFIX}/conda-bld/
```
Comment (MetRonnie) [inline docs/howto/create-a-release.md:232]: Oops, just spotted, shouldn't this be
```diff
-conda install -c ${CONDA_PREFIX}/conda-bld/
+conda install -c ${CONDA_PREFIX}/conda-bld/ $PACKAGE_NAME
```

**Pull request merged**